### PR TITLE
docs: update doc for edgex2.2 upgrade

### PR DESCRIPTION
**Pull request by vli11**
Signed-off-by: Valina Li <valina.li@intel.com>

### CODE REVIEW

File: docs/automated-checkout-services/device_services.html [hunk 1]
```diff
@@ -887,32 +887,32 @@ <h3 id="controller-board-physical-device-functionality">Controller board physica
 <p>Therefore, if you have multiple serial devices plugged into your system, please manually edit the <code>docker-compose.yml</code> file to mount your <code>/dev/ttyACM_X_</code> appropriately. You may want to explore creating udev rules to enforce TTY consistency.</p>
 <h3 id="controller-board-apis">Controller board APIs</h3>
 <hr />
-<h4 id="put-httplocalhost48098apiv1devicenameds-controller-boardcommandlock1"><code>PUT</code>: <code>http://localhost:48098/api/v1/device/name/ds-controller-board/command/lock1</code></h4>
+<h4 id="put-httplocalhost48098apiv1devicenameds-controller-boardcommandlock1"><code>PUT</code>: <code>http://localhost:48098/api/v2/device/name/ds-controller-board/command/lock1</code></h4>
```
Comment (jim-wang-intel) [inline docs/automated-checkout-services/device_services.html:890]: is this html generated from a tool?  if not, could we make this long line as separate lines for more readable?
Similar thing applied to this whole html having a long line.

File: docs/phases/phase1.html [hunk 2]
```diff
@@ -735,7 +735,7 @@ <h3 id="1-stock-the-cooler-with-inventory">1. Stock the cooler with inventory</h
 
 <p>Then open the door, and close it afterwards, while waiting approximately 3-4 seconds between each event using the following 2 commands.</p>
 <p>The following command makes a REST API call to the <code>ds-controller-board</code> service to open the door (no response body expected) <strong><em>(time sensitive)</em></strong>:</p>
-<div class="codehilite"><pre><span></span><code>curl -X PUT -H <span class="s2">&quot;Content-Type: application/json&quot;</span> -d <span class="s1">&#39;{&quot;setDoorClosed&quot;:&quot;0&quot;}&#39;</span> http://localhost:48082/api/v1/device/name/ds-controller-board/command/setDoorClosed
+<div class="codehilite"><pre><span></span><code>curl -X PUT -H <span class="s2">&quot;Content-Type: application/json&quot;</span> -d <span class="s1">&#39;{&quot;setDoorClosed&quot;:&quot;0&quot;}&#39;</span> http://localhost:48082/api/v2/device/name/ds-controller-board/command/setDoorClosed
```
Comment (jim-wang-intel) [inline docs/phases/phase1.html:738]: similar thing for readability in this file

File: docs/automated-checkout-services/device_services.html (hunk 1 above)
Comment (brian-intel) [reply docs/automated-checkout-services/device_services.html:890]: @jim-wang-intel Yes, this was originally generated by a tool and should be regenerated.

**Comment by vli11** [comment]
doc on-hold for now, need to update after few refactoring is done

File: docs_src/phases/phase1.md [hunk 3]
```diff
@@ -12,17 +12,15 @@ The documentation in [phase 2](./phase2.md) and [phase 3](./phase3.md) will disc
...
-    - A maintenance worker resolves the issue
+- [Phase 1 - Simulation Mode](#phase-1---simulation-mode)
+  - [Overview](#overview)
+    - [Scenarios](#scenarios)
+  - [Getting started](#getting-started)
+  - [Walkthrough of scenarios](#walkthrough-of-scenarios)
+    - [1. Stock the cooler with inventory](#1-stock-the-cooler-with-inventory)
+    - [2. Purchase from the cooler as a customer](#2-purchase-from-the-cooler-as-a-customer)
+    - [3. The cooler requires maintenance](#3-the-cooler-requires-maintenance)
+  - [Summary](#summary)
```
Comment (brian-intel) [inline docs_src/phases/phase1.md:23]: To keep consistency with the other phase.md file formats we should revert the changes.

**Review by brian-intel — CHANGES_REQUESTED** [review]

**Review by brian-intel — APPROVED** [review]

**Review by joshua-silverio — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]

**Review by brian-intel — APPROVED** [review]

**Review by marcpfuller — APPROVED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM
